src/OpenLoco/src/Paint/Paint.h

```diff
@@ -122,12 +122,8 @@ namespace OpenLoco::Paint
 
     struct PaintStructBoundBox
     {
-        int16_t x;
-        int16_t y;
-        int16_t z;
-        int16_t xEnd;
-        int16_t yEnd;
-        int16_t zEnd;
+        World::Pos3 mins;
+        World::Pos3 maxs;
```

Comment:
Guess `min` and `max` are reserved names, right? Due to the `s`, this reads as plural, though.

Reply:
Yeah min/max are often macros, I just went with what Valve has:
![image](https://github.com/user-attachments/assets/b123d9dd-a788-425b-b2e0-731f5d66e3b7)